docs/source/features/version_control/draft.rst

```diff
@@ -109,6 +109,24 @@ saved into this commit.
    draft.commit(f"{COMMIT_TITLE}")
    draft.commit(f"{COMMIT_TITLE}", f"{COMMIT_DESCRIPTION}")
 
+.. attention::
+   SDK does not automatically update the ``dataset.HEAD`` after committing the draft. Therefore,
+   the dataset may be one commit behind the server. In addition, consider the concurrent operation
```

Comment:
Only one commit behind？Could there be more commits behind?
`consider the concurrent operation`? Rephrase this for better understanding?